boards/mediatek/mt8186/mt8186_adsp.dts

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2024 The ChromiumOS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+
+/dts-v1/;
+/ {
+
```

Comment:
missing indent below here

Reply:
This comment got made earlier and the response was that it was intentional, for the same reason you don't indent inside a `namespace { }` directive in C++.   I thought you were OK with that?

Reply:
I've only ever seen it be indented on boards e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf52840dk/nrf52840dk_nrf52840.dts I can't say I've seen a board where it's not indented, and for the C++ code bases I have the namespace {} is indented

Reply:
So you're not OK with that then?  Will fix.